ulib/Makefile

```diff
@@ -50,6 +50,9 @@ ulib-in-fsharp:
 	+$(MAKE) -f Makefile.extract.fsharp all-fs
 	+$(MAKE) -C fs/VS
 
+install-ulib-in-fsharp: ulib-in-fsharp
+	cp fs/bin/Release/netstandard2.0/ulibfs.* ../bin/
+
```

Comment:
Can you please say more, what changes in this PR prompted this new target to copy some files?

Reply:
That's to make example `hello` works. Maybe that's wrong location, but I cannot find who copy `ulib.dll` to `bin` folder, without that F# example `hello` does not work on clean machine.

Reply:
Ok, thanks for clarifying, let's wait for @mateuszbujalski to chime in.

Reply:
See https://github.com/FStarLang/FStar/pull/2513/files#r848888382. 
I wonder if it makes sense to try to copy pdb when you take a release build anyway.

Reply:
I thought about having separate changes
- mechanical changes (like this one where I just bump tooling)
- some other changes to simplify improve infra.

i.e. this is just foundation for future work. I can make changes here if you like.

Reply:
I restore functionality per your comment. Overlook and overthink things.